### ISSUE TITLE
Can I change global theme

### ISSUE THREAD

**Issue by HoxinhLuo** [honghaier-game/TKinterDesigner#45]
Can I change global theme of software in V1.6.8 in windows x64 due to the current theme is dark which I need to change to light. Thanks.

**Comment by honghaier-game** [comment]
yes,you only need open the dialog of 'setting', it is in the  first item of top menu,and if you can see a  gear icon, click it to see the color theme settings.
